providers/alicloud/alicloud_service.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2018 The Terraformer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package alicloud
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraform_utils"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
```

Comment:
Please use github.com/aliyun/alibaba-cloud-sdk-go direct and not package from provider.
Because now we need vendor all alicloud provider and we compile all sdk packages from alicloud
https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/alicloud/connectivity/client.go#L5-L37

Reply:
I dont see whats wrong with that because eventually all these would have to be imported anyways. 

If I use  `github.com/aliyun/alibaba-cloud-sdk-go` direct I will be just copying over the `github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity` package.

Reply:
Also having a single source of truth for this is pretty useful
https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/alicloud/connectivity/regions.go

Reply:
2 issues:
1. We imported  all terraform-provider-alicloud dependency(like github.com/valyala/fasthttp I don't know why alicloud provider need this?)
2. Connectivity package import all alibaba-cloud-sdk-go packages, it's will be compile in binary, include packages not used by terraformer.

Reply:
`terraform-provider-alicloud` is not the reason for increased binary size. The `alicloud-go-sdk` is fat.

Here is the output of `goweight`
https://gist.github.com/Ghost---Shadow/e57a21bd2592b1d38d76131d834e8838

Reply:
I will check, again